modules/programs/freetube.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isLinux;
+  # FIXME-REDOX(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isRedox;
+  # FIXME-HURD(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isHurd;
+  # FIXME-FREEBSD(Krey): Integrate
+  #inherit (pkgs.stdenv.hostPlatform) isFreebsd;
+  inherit (pkgs.stdenv.hostPlatform) isCygwin;
```

Comment:
We don't support these platforms at this time so this can be removed.
```suggestion
```

Reply:
NixOS has logic for them so i think it could be integrated in case home-manager decides to support them?

(i like to make my code crossplatform whenever possible)